gp2gp-translator/src/test/resources/xml/Patient/patient_example.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think this should be just patient here. This EhrExtract element is not xsd schema valid